quint/src/repl.ts

```diff
@@ -362,44 +376,41 @@ function loadFromFile(out: writer, state: ReplState, filename: string): boolean
     const newState = { ...state }
     const modulesAndRepl = data.split(replBegin)
     // whether an error occurred at some step
-    let isError = false
     newState.moduleHist = modulesAndRepl[0]
     if (modulesAndRepl.length > 1) {
       // found a REPL session
       const defsAndExprs = modulesAndRepl[1].split(replEnd)
 
       // save the definition history
       newState.defsHist = defsAndExprs[0]
+
```

Comment:
```suggestion
     // evaluate a dummy definition, in order to compile the definitions from the file
```

Shall we use `val _ = true` in the future? I guess, `_` would not work as expected now, but we should make it work at some point.